modules/builds-identifying-image-change-triggers.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * builds/triggering-builds-build-hooks.adoc
+
+[id="builds-image-change-trigger-identification_{context}"]
+= Image change trigger identification
+
+As a developer, if you have configured image change triggers, you can identify which image change initiated the last build.
+
+To accomplish this, you must identify elements in your build configuration's specification and status that are related to image change triggers.
+
+This way, you can use the timestamp in `buildConfig.status.imageChangeTriggers` to identify the most recent build. Then you can use the name and namespace of the image stream that triggered this build to find the corresponding image change trigger in the `buildConfig.spec.triggers`.
+
+
+== Image change trigger elements in the specification
+
+In your build configuration specification, `buildConfig.spec.triggers` is an array of build trigger policies, `BuildTriggerPolicy`.
+
+Each `BuildTriggerPolicy` has a `type` field and set of pointers fields, where each pointer field corresponds to one of the allowed values for the `type` field. As such, only one pointer field can be set for a given `BuildTriggerPolicy`.
+
+So for image change triggers, the value of `type` is `ImageChange`.
+
+Then, the `imageChange` field is the pointer to an `ImageChangeTrigger` object. So this will be set. It has the following fields:
+
+* `lastTriggeredImageID`: This field is deprecated in {product-title} 4.8, but is still being set. It will be ignored in a future release. It contains the resolved image reference for the `ImageStreamTag` when the last build was triggered from this `BuildConfig`.
+* `paused`: This field is used to temporarily disable this particular image change trigger.
+* `from`: This field is used to reference the `ImageStreamTag` that drives this image change trigger. Its type is the core Kubernetes type, `OwnerReference`. The `from` field has the following fields of note:
```

Comment:
" The `from` field has the following fields of note:"  Formatting question, should the next couple of bullets be formatted as second level under this bullet?

Reply:
Thanks. Updated.